src/components/lenker.tsx

```diff
@@ -221,6 +221,9 @@ function Lenker(props: Props) {
                         <Lenke href={`${modiacontextholderUrl}/redirect/salesforce`} target="_blank">
                             Salesforce
                         </Lenke>
+                        <Lenke href="https://nav-arbeidsmarkedstiltak.intern.nav.no/" target="_blank">
```

Comment:
For at vi skal få riktig lenke når dekoratøren kjører i dev burde det være (antar at dere har et dev miljø også 😄): 

```suggestion
                        <Lenke href={`https://nav-arbeidsmarkedstiltak${naisInternNavDomain}/`} target="_blank">
```